frontend/src/components/Dialogs/Profile.tsx

```diff
@@ -206,6 +210,22 @@ const ProfileDialog = ({
 
           <Divider/>
 
+          <Grid container>
+            <Grid item>
+              <FormControlLabel
+                labelPlacement="start"
+                label={`${t("Use stealth invoices")}`}
```

Comment:
The translation key `"use stealth invoices":"{same but in another language}"` is now available. Would be ideal to add a new key to `en.json` so future translations include it. It can be added at the end of "Bottom Bar", around here:
https://github.com/Reckless-Satoshi/robosats/blob/efd89756e41820cd0a1cc26105d23a7300094f84/frontend/static/locales/en.json#L229-L230